src/module/vector.jl

```diff
@@ -138,17 +138,33 @@ function *(a::svector{spoly{T}}, b::spoly{T}) where T <: AbstractAlgebra.RingEle
    return svector{spoly{T}}(R, a.rank, s)
 end
 
+function *(a::svector{Singular.spluralg{T}}, b::Singular.spluralg{T}) where T <: AbstractAlgebra.RingElem
+   base_ring(a) != parent(b) && error("Incompatible base rings")
+   R = base_ring(a)
+   s = GC.@preserve a b R libSingular.pp_Mult_qq(a.ptr, b.ptr, R.ptr)
+   return svector{spoly{T}}(R, a.rank, s)
+end
+
 function (a::spoly{T} * b::svector{spoly{T}}) where T <: Nemo.RingElem
    base_ring(b) != parent(a) && error("Incompatible base rings")
    R = base_ring(b)
    s = GC.@preserve a b R libSingular.pp_Mult_qq(a.ptr, b.ptr, R.ptr)
    return svector{spoly{T}}(R, b.rank, s)
 end
 
+function *(b::Singular.spluralg{T}, a::svector{Singular.spluralg{T}}) where T
+   base_ring(a) != parent(b) && error("Incompatible base rings")
+   R = base_ring(a)
+   s = GC.@preserve a b R libSingular.pp_Mult_qq(a.ptr, b.ptr, R.ptr)
```

Comment:
Same question here: is this the right function to call here?

Reply:
Yes, pp_Mult_qq (and most other p_*,pp_* routines) work for all types of Singular rings.